community/kernel/src/main/java/org/neo4j/kernel/impl/api/index/IndexingService.java

```
@@ -337,6 +347,32 @@ public void start() throws Exception
         state = State.RUNNING;
     }
 
+    private void performRecoveredIndexDropActions()
+    {
+        indexesToDropAfterCompletedRecovery.values().forEach( index ->
+        {
+            try
+            {
+                index.drop();
+            }
+            catch ( Exception e )
+            {
+                // This is OK to get during recovery because the underlying index can be in any unknown state
+                // while we're recovering. Let's just move on to closing it instead.
+                try
+                {
+                    index.close();
+                }
+                catch ( IOException closeException )
+                {
+                    // This is OK for the same reason as above
+                    e.addSuppressed( closeException );
```

If we are swallowing exception here we don't need to `addSuppressed`.

---

That's right. I forgot to remove that when finalizing this fix.